Add direct link to auto discovery docs in mysql/postgres aurora setup (#23110)
DataDog/documentation: content/en/database_monitoring/setup_postgres/aurora.md

2. [Restart the Agent][2].

> **The Datadog Agent supports auto-discovering all aurora endpoints in a cluster.** In order to do this, please [follow this guide.][14]

Reviewer: ```suggestion
**The Datadog Agent supports [Autodiscovery][14] of all Aurora endpoints in a cluster.** 
```

Same with this one
